core/src/test/java/org/mapfish/print/processor/map/CreateMapProcessorWmtsBufferTest.java

```diff
@@ -76,7 +76,7 @@ public void testExecute() throws Exception {
     List<URI> layerGraphics = (List<URI>) values.getObject("layerGraphics", List.class);
     assertEquals(1, layerGraphics.size());
 
-    new ImageSimilarity(new File(layerGraphics.get(0)))
-        .assertSimilarity(getFile(BASE_DIR + "/expectedSimpleImage.png"), 0);
+    new ImageSimilarity(getFile(BASE_DIR + "/expectedSimpleImage.png"))
```

Comment:
Good idea. Thanks
Have you double checked if this was the only place where we called it wrong ?

Reply:
As I see yes :-)